rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/selection/ScriptExecutor.java

```diff
@@ -90,6 +91,7 @@ private Node executeScripts() {
                 logger.info(rmnode.getNodeURL() + " : " + script.hashCode() + " executing");
                 try {
                     ScriptResult<Boolean> scriptResult = rmnode.executeScript(script, criteria.getBindings());
+                    atLeastOneScriptExecuted = true;
```

Comment:
this should be put before, an exception (timeout) will shortcut this affectation.

Reply:
Yes ok, done.